src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+    private String color;
+
+    public IsoscelesTrapezoid(Color color, double base1, double base2, double height) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables for 'color', 'base1', 'base2', and 'height' and pass them on for better code readability.